src/components/Footer.tsx

```diff
@@ -37,47 +37,59 @@ const techStack: TechStack[] = [
 export default function Footer() {
   return (
     <footer className="bg-[#0C111C] py-16">
-      <div className="mx-auto flex max-w-56 flex-col items-center md:max-w-4xl">
-        <span className="mb-2 text-sm font-medium leading-8 text-white md:text-3xl">
-          Built With
-        </span>
+      <div className="mx-auto flex max-w-56 flex-col items-center gap-y-10 md:max-w-4xl">
+        <a
+          href="https://github.com/Learn-Build-Teach/deals-for-devs"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="font-lg mt-8 flex items-center gap-x-2 rounded-md bg-gray-800 px-4 py-2 text-sm text-white transition-transform hover:scale-105 hover:bg-gray-700 "
+        >
+          <FaGithub />
+          <span>View on GitHub</span>
+        </a>
+        <div>
+          <p className="mb-4 text-center text-sm font-light leading-8 text-white md:text-xl">
+            Built With
+          </p>
+          <div className="flex flex-wrap items-center justify-center gap-4 md:gap-10">
+            {techStack.map((sponsor, index) => (
+              <a
+                key={index}
+                href={sponsor.Url}
+                target="_blank"
+                aria-label={sponsor.alt}
+                className="transition-transform hover:scale-105 "
+              >
+                <Icon
+                  id={sponsor.id}
+                  size={sponsor.size}
+                  className={sponsor.className}
+                />
+              </a>
+            ))}
 
-        {/* built with logos */}
-        <div className="flex flex-wrap items-center justify-center gap-4 md:gap-10">
-          {techStack.map((sponsor, index) => (
-            <a
-              key={index}
-              href={sponsor.Url}
-              target="_blank"
-              aria-label={sponsor.alt}
-            >
-              <Icon
-                id={sponsor.id}
-                size={sponsor.size}
-                className={sponsor.className}
+            {/* clerk was too difficult to make into a sprite */}
+            <a href="https://clerk.com/" target="_blank">
+              <Image
+                src="/icons/clerk.svg"
+                width={159}
+                height={49}
+                className="h-6 w-20 transition-transform hover:scale-105 "
```

Comment:
nitpick: Could you remove the space at the end.

Reply:
Got it!